resources/v2/08_manufacturers.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE manufacturers
+( id  SERIAL PRIMARY KEY,
```

Comment:
format this, either use prettier or you can format it using: https://codebeautify.org/sqlformatter

Reply:
same for all the raw sql files